lib/lightning/attempts/attempt.ex

```diff
@@ -56,7 +56,10 @@ defmodule Lightning.Attempt do
     has_one :workflow, through: [:work_order, :workflow]
 
     has_many :log_lines, LogLine
-    many_to_many :runs, Run, join_through: AttemptRun
+
+    many_to_many :runs, Run,
+      join_through: AttemptRun,
+      preload_order: [asc: :finished_at]
```

Comment:
Shouldn't this be `:started_at`? or perhaps `asc: :finished_at, asc: :started_at` because a fun can be either in progress or 'lost' where the finished at (afaik) isn't ever going to be set?

@taylordowns2000 care to comment?

Reply:
yes, great call @stuartc . @midigofrank , please make this `asc: :started_at`